ansible/roles/nginx/vars/motech.yml

```diff
@@ -20,3 +20,6 @@ nginx_sites:
      - name: /supply-dhis
        proxy_pass: http://motech-main.internal.commcarehq.org:8888/supply-dhis
        proxy_read_timeout: 180s
+     - name: /cdc-moz
+       proxy_pass: http://motech3.internal.commcarehq.org:8080/cdc-moz
```

Comment:
Uh, yeah confusing @biyeun but I think this is "correct" in the sense that this points to the right machine, it's just that the DNS @sheelio made didn't conform to the convention we've been using. I don't think it's a huge deal but we can easily change that.